### PR TITLE
Random tidbits

### PR DESCRIPTION
**Pull request by andrewconner**

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html [hunk 1]
```diff
@@ -8,8 +8,8 @@
   <div
     ng-if="shouldShowSignupBanner()"
     kf-generic-banner
-    thumb="{{membership.invite.inviter|pic}}"
-    text="{{membership.invite.inviter|name}} invited you to become a member of this team."
+    thumb="{{viewer.invite.inviter|pic}}"
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html:11]: @cvializ FYI, was broken

File: kifi-backend/modules/common/app/com/keepit/model/OrganizationInfo.scala [hunk 2]
```diff
@@ -63,15 +63,17 @@ object OrganizationViewerInfo {
 
 case class OrganizationInviteInfo(
   inviter: BasicUser,
+  email: Option[EmailAddress],
   lastInvited: DateTime)
 object OrganizationInviteInfo {
   implicit val internalFormat: Format[OrganizationInviteInfo] = (
     (__ \ 'inviter).format[BasicUser] and
+    (__ \ 'email).formatNullable[EmailAddress] and
```
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/OrganizationInfo.scala:71]: @ryanpbrewster FYI

File: kifi-backend/modules/shoebox/angular/src/layout/main/MainCtrl.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ angular.module('kifi')
     var importBookmarksMessageEvent;
     var bannerMessages = {
       0: 'Welcome back!',
+      1: 'Email confirmed. Thanks!',
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/layout/main/MainCtrl.js:16]: @eishay FYI

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html (hunk 1 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.tpl.html:11]: nice catch

File: kifi-backend/modules/common/app/com/keepit/model/OrganizationInfo.scala (hunk 2 above)
Comment (ryanpbrewster) [inline kifi-backend/modules/common/app/com/keepit/model/OrganizationInfo.scala:71]: :panda_face: :clock2:

**Pull request merged**